src/main/java/com/fundation/webservice/controller/VideoResponse.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2019 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+package com.fundation.webservice.controller;
+
```

Comment:
Is it necessary this space?

Reply:
Yes. Actually, this space separates the package section from the import section.

Reply:
ok